homework_11/library.py

```diff
@@ -0,0 +1,130 @@
+"""Library"""
+
+
+class Book:
+    """Класс книга"""
+    def __init__(
+        self,
+        name: str,
+        author: str,
+        page_count: int,
+        isbn: int,
+        is_reserved: bool
+    ):
+        self.name = name
+        self.author = author
+        self.page_count = page_count
+        self.isbn = isbn
+        self.is_reserved = is_reserved
+
+
+class User:
+    """Класс пользователя"""
+    def __init__(self, first_name: str, last_name: str):
+        self.first_name = first_name
+        self.last_name = last_name
+
+        self.reserved_books: list[Book] = []
+        self.taken_books: list[Book] = []
+
+    def reserve_book(self, book: Book):
```

Comment:
Need to specify what type method\function will return
```suggestion
    def reserve_book(self, book: Book) -> bool:
```